lms/djangoapps/shoppingcart/tests/test_reports.py

```diff
@@ -229,7 +233,7 @@ def test_purchased_csv(self):
         report = initialize_report("itemized_purchase_report", self.now - self.FIVE_MINS, self.now + self.FIVE_MINS)
         csv_file = StringIO()
         report.write_csv(csv_file)
-        csv = csv_file.getvalue()
+        csv = csv_file.getvalue() if six.PY3 else csv_file.getvalue().decode('utf-8')
```

Comment:
@feanil Im a bit confused regarding how can i update the upstream function. As its coming from StringIo.